src/postprocessing.py

```diff
@@ -81,11 +81,11 @@ def erode_image(mask, erode_selem_size):
     """Erode mask.
 
     Args:
-        mask (numpy.ndarray): mask of shape (H x W) or set of masks of shape (C x H x W)
-        erode_selem_size (int): size of rectangle structuring element used for erosion
+        mask (numpy.ndarray): Mask of shape (H x W) or set of masks of shape (C x H x W).
```

Comment:
@taraspiotr do you think that  `set` could be confusing and `list of masks` is better?

Reply:
@jakubczakon `list` and `set` are both python built-ins so maybe `multiple masks`?